flash/core/utilities/imports.py

```diff
@@ -94,3 +94,19 @@ def _compare_version(package: str, op, version) -> bool:
 _VIDEO_AVAILABLE = _PYTORCHVIDEO_AVAILABLE
 _IMAGE_AVAILABLE = _TORCHVISION_AVAILABLE and _TIMM_AVAILABLE and _KORNIA_AVAILABLE
 _SERVE_AVAILABLE = _FASTAPI_AVAILABLE and _PYDANTIC_AVAILABLE and _CYTOOLZ_AVAILABLE and _UVICORN_AVAILABLE
+
+_IMAGE_TESTING = False
+_VIDEO_TESTING = False
+_TABULAR_TESTING = False
+_TEXT_TESTING = False
+_IMAGE_STLYE_TRANSFER_TESTING = False
+_SERVE_TESTING = False
+
+if "FLASH_TEST_TOPIC" in os.environ:
+    topic = os.environ["FLASH_TEST_TOPIC"]
+    _IMAGE_TESTING = topic == "image"
+    _VIDEO_TESTING = topic == "video"
+    _TABULAR_TESTING = topic == "tabular"
+    _TEXT_TESTING = topic == "text"
+    _IMAGE_STLYE_TRANSFER_TESTING = topic == "image_style_transfer"
+    _SERVE_TESTING = topic == "serve"
```

Comment:
Can we move this while part to the tests package root? This is not necessary to include when shipping the package

Reply:
Done :smiley: